MapboxNavigation/RouteMapViewController.swift

```diff
@@ -589,6 +589,25 @@ extension RouteMapViewController: NavigationViewDelegate {
     
     //MARK: InstructionsBannerViewDelegate
     func didTapInstructionsBanner(_ sender: BaseInstructionsBannerView) {
+        repeatInstructions()
+    }
+    
+    func didDragInstructionsBanner(_ sender: BaseInstructionsBannerView) {
+        displayPreviewInstructions()
+    }
+    
+    private func repeatInstructions() {
```

Comment:
Per https://github.com/mapbox/mapbox-navigation-ios/issues/1138#issuecomment-370938050, we discussed that the first two steps would be implemented first. No need to do anything around repeating the instruction yet.

Reply:
Yes. Will revert that work.

Reply:
@bsudekum done ✅. This private function will just live here till we are ready to plug it in a delegate or gesture recognizer

Reply:
@vincethecoder I think we should remove it and add it when it's time to add this feature.

Reply:
Ok